ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -400,28 +402,50 @@
         show-footer? (if (get-in app [:transport-service ::t-service/id])
                        (ts/is-service-owner? app)
                        true)]
-
-    (when true ;show-footer? - Take owner check away for now
-      [:div.row
-       (when (not (form/can-save? data))
-         [ui/card {:style {:margin-bottom "1rem"}}
-          [ui/card-text {:style {:color "#be0000" :padding-bottom "0.6rem"}} (tr [:form-help :publish-missing-required])]])
-
-       (if published?
-         ;; True
-         [buttons/save {:on-click #(e! (ts/->SaveTransportService schemas true))
-                        :disabled (not (form/can-save? data))}
-          (tr [:buttons :save-updated])]
-         ;; False
-         [:span
+    [:div
+     ;; Show brokering dialog
+     (when (get-in app [:transport-service :show-brokering-service-dialog?])
+       [ui/dialog
+        {:id "brokering-service-dialog"
+         :open true
+         :title (tr [:dialog :brokering-service :title])
+         :actions [(r/as-element
+                     [ui/flat-button
+                      {:label (tr [:dialog :brokering-service :cancel])
+                       :primary true
+                       :on-click #(e! (ts/->SelectBrokeringService false))}])
+                   (r/as-element
+                     [ui/raised-button
+                      {:id "confirm-brokering-service"
+                       :label (tr [:dialog :brokering-service :ok])
+                       :icon (ic/action-check-circle)
+                       :secondary true
+                       :primary true
+                       :on-click #(e! (ts/->SelectBrokeringService true))}])]}
+        [:p (tr [:dialog :brokering-service :body])]])
+
+     ;show-footer? - Take owner check away for now
+     (when true
+       [:div.row
+        (when (not (form/can-save? data))
+          [ui/card {:style {:margin-bottom "1rem"}}
+           [ui/card-text {:style {:color "#be0000" :padding-bottom "0.6rem"}} (tr [:form-help :publish-missing-required])]])
```

Comment:
Colour codes should be `def`'d in `colors.clj` and flow through a semantical `def`.
For an example please take a look on `(def warning red-basic)`.

Similar for the `:padding-bottom`.

All in all a recommended pattern could be here to have a view or card-specific `def` style in a new transport-service.cljs or append to base.cljs, then just apply that def´d style here to the card element.
That style then would be composed like comments above propose.
Result would be that updating styles via def is straightforward and maintainable.

Reply:
Agreed. However I'm working on a branch with several ui /warning fixes and I'll take care of this in that branch.
(this is old code and I didn't change it in this pr)